lesson14/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css"  />
+    <script src="./main.js" defer></script>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <div id="js-modal-wrapper">
+        <button id="js-open-modal-button" class="button">Open Modal</button>
+        <div id="js-overlay" class="overlay"></div>
+        <div id="js-modal" class="modal">
+          <div id="js-close-button" class="close-button">X</div>
+          <p class="modal-title">Please enter a number</p>
+          <div class="flexbox">
+            <input type="text" id="js-number-input" class="input-field">
+            <p id="js-validation-number" class="item modal-validation">Input value must be a number</p>
+            <p id="js-validation-empty" class="item modal-validation">Input field cannot be empty</p>
```

Comment:
comment
I've never seen a pattern where two error messages are prepared on the HTML side, 
so I wondered if it would be okay to prepare one p for the error message and replace it. 
But this is my opinion, so it might be a matter of preference.

If you have any references to it, let me know!

Reply:
Thank you for the comment, I was not aware that it is hard-coding and would be able to handle by JS DOM as the previous lessons. 
Updated validation message function 
Please check the commit
10d6a13925dbfcfc0fd346260585132aa1dd91ed